llama_hub/bagel/README.md

```diff
@@ -0,0 +1,35 @@
+# Bagel Loader
+
+The Bagel Loader returns a set of texts corresponding to query embeddings or query texts retrieved from a BagelDB.
+The user initializes the loader with a BagelDB. They then pass in a query vector or a query text along with optional query parameters like metadata, where, where documents and include.
+
+## Usage
+
+Here's an example usage of the BagelReader.
+
+```python
+from llama_index import download_loader
```

Comment:
nit: canonical is now `from llama_hub.bagel.base import BagelReader` (no need for download_loader)

Reply:
Changed in ReadMe..

Thanks